src/50_Samples_%26_Templates/Product.html

```diff
@@ -544,21 +550,21 @@ <h2 class="m1">Apple</h2>
         <div class="items-center flex">
           <label for="size">Size:</label>
           <amp-selector name="size" layout="container"
-                  on="select:AMP.setState({selectedSize: event.targetOption, selectedColor: selectedColor ? selectedColor : 'green'})"
-                  [selected]="(product[selectedColor || 'green'].sizes[selectedSize || 'S'] != 'unavailable')
-                      ? selectedSize
-                      : product[selectedColor || 'green'].defaultSize">
+                  on="select:AMP.setState({ product: {selectedSize: event.targetOption, selectedColor: selectedColor ? selectedColor : 'green'}})"
```

Comment:
Is this the size selector? If so, we shouldn't need to change the color here 
right?